crates/ruff_python_formatter/src/options.rs

```diff
@@ -207,35 +207,7 @@ pub enum QuoteStyle {
     Single,
     #[default]
     Double,
-}
-
-impl QuoteStyle {
-    pub const fn as_char(self) -> char {
-        match self {
-            QuoteStyle::Single => '\'',
-            QuoteStyle::Double => '"',
-        }
-    }
-
-    #[must_use]
-    pub const fn invert(self) -> QuoteStyle {
-        match self {
-            QuoteStyle::Single => QuoteStyle::Double,
-            QuoteStyle::Double => QuoteStyle::Single,
-        }
-    }
-}
-
-impl TryFrom<char> for QuoteStyle {
-    type Error = ();
-
-    fn try_from(value: char) -> std::result::Result<Self, Self::Error> {
-        match value {
-            '\'' => Ok(QuoteStyle::Single),
-            '"' => Ok(QuoteStyle::Double),
-            _ => Err(()),
-        }
-    }
+    Preserve,
```

Comment:
We need to update the documentation of the `FormatOption::quote_style` field to make the change reflect on our website:

https://github.com/astral-sh/ruff/blob/578ddf1bb1e8edef50cda9981744ba5fe825867d/crates/ruff_workspace/src/options.rs#L2804-L2829